crates/db_views_actor/src/community_view.rs

```diff
@@ -183,12 +183,33 @@ impl<'a> CommunityQuery<'a> {
     }
 
     match self.sort.unwrap_or(SortType::Hot) {
+      SortType::Hot => query = query.order_by(community_aggregates::hot_rank.desc()),
+      SortType::Active => query = query.order_by(community_aggregates::hot_rank.desc()),
```

Comment:
It would be easier to read if you put `use SortType::*` before the match. Then you can do `Hot | Active => ...` in cases where the same statement is used for multiple cases.

Reply:
K done.